doc/source/whatsnew/v1.1.5.rst

```diff
@@ -14,8 +14,7 @@ including other versions of pandas.
 
 Fixed regressions
 ~~~~~~~~~~~~~~~~~
--
--
+- Fixed regression in :meth:`DataFrame.to_hdf` was not dropping missing rows with ``dropna=True`` (:issue:`35719`)
```

Comment:
this has been quite a while, move this to 1.2

Reply:
Done